exec/java-exec/src/main/codegen/data/DateIntervalFunc.tdd

```diff
@@ -18,4 +18,13 @@
     {intervals: ["Interval", "IntervalDay", "IntervalYear", "Int", "BigInt"] },
     {truncInputTypes: ["Date", "TimeStamp", "Time", "Interval", "IntervalDay", "IntervalYear"] },
     {truncUnits : ["Second", "Minute", "Hour", "Day", "Month", "Year", "Week", "Quarter", "Decade", "Century", "Millennium" ] },
+
+    {emptyStringToDateConversion: [
```

Comment:
`emptyStringToDateConversion` -> `VarCharToDate`? Not only empty strings are converted to Date